api/models/user.py

```diff
@@ -0,0 +1,51 @@
+"""
+User class. Contains the relevant methods for the user class
+"""
+import re
+
+class User():
+    '''a base pssanger class from which a driver can be made'''
+
+    def __init__(self, name, phone, password, confirm_p):
+        ''' constructor method to give a user its attributes'''
+        self.name = name
+        self.phone = phone
+        self.password = password
+        self.confirm_p = confirm_p
```

Comment:
I think using `confirm_password` instead of `confirm_p` sounds better
`confirm_p` could be interpreted as *confirm phone* :smile:

Reply:
noted, i will make the change in that variable name. thanks